src/xunit.netcore.extensions/PlatformID.cs

```diff
@@ -9,10 +9,11 @@ namespace Xunit
     [Flags]
     public enum PlatformID
     {
-        Windows = 1,
+        Any = ~0,
+        AnyUnix = FreeBSD | Linux | OSX,
+        FreeBSD = 8,
         Linux = 2,
         OSX = 4,
-        AnyUnix = Linux | OSX,
-        Any = Windows | Linux | OSX
+        Windows = 1
```

Comment:
I'd prefer to leave this not sorted alphabetically.  If the list eventually grows long, we can consider it, but with so few items, I prefer the ordering to be the way it was, just with FreeBSD inserted between OSX and AnyUnix.